rafx-framework/Cargo.toml

```diff
@@ -27,6 +29,8 @@ profiling = "0.1.9"
 serde = { version = "1", features = ["serde_derive"] }
 serde_bytes = "0.11"
 glam = { version = "0.13.1", features = [ "serde" ] }
+parking_lot = "0.11.1"
```

Comment:
another note to myself to come back and look at exactly why parking_lot is needed, if we already pull it in as a dependency, or pull in an alternative to it that could fill the same role.